packages/v4/patternfly-docs/content/design-guidelines/components/table/table.md

```diff
@@ -174,10 +175,4 @@ Lists and tables should be placed in the body of a page. The width should be set
 
```

Comment:
Also, I notice there's a random "Using lists and tables in a page" section - wonder if we could put it inside the usage section instead?